### PR TITLE
Add scams from ChainPatrol

### PR DESCRIPTION
**Pull request by NikitaVr**
## Scam links
- [starknet-zealy.quest](https://starknet-zealy.quest)
- [starknetcliam.click](https://starknetcliam.click)

---

## Scam links
- [linea-build.app](https://linea-build.app)

---

## Scam links
- [zksyncprotcol.com](https://zksyncprotcol.com)

---

## Scam links
- [arbitrum.free-nft.org](https://arbitrum.free-nft.org)
- [arb-check.claim-web3.run](https://arb-check.claim-web3.run)
- [arbitrum-4.xyz](https://arbitrum-4.xyz)
- [arbitrumsurvey.net](https://arbitrumsurvey.net)
- [arbitrum-6.xyz](https://arbitrum-6.xyz)

---

## Scam links
- [consensys.free-nft.org](https://consensys.free-nft.org)

---

## Scam links
- [tokenmask.io](https://tokenmask.io)

---

## Scam links
- [zksync.free-nft.org](https://zksync.free-nft.org)

---

## Scam links
- [taikoclaim.click](https://taikoclaim.click)

---

## Scam links
- [explore-sync.com](https://explore-sync.com)

---

## Scam links
- [starknet-pro.com](https://starknet-pro.com)

---

## Scam links
- [claim.starknet-token.xyz](https://claim.starknet-token.xyz)
- [starknet-token.xyz](https://starknet-token.xyz)

---

## Scam links
- [reg-manta.network](https://reg-manta.network)

---

## Scam links
- [zurmarket-v1vercel.app](https://zurmarket-v1vercel.app)

---

## Scam links
- [multichain-webinchpd.com](https://multichain-webinchpd.com)

---

## Scam links
- [thegraphairdrop.net](https://thegraphairdrop.net)
- [grt.100pad.top](https://grt.100pad.top)
- [yearn.100pad.top](https://yearn.100pad.top)
- [dappradar.land](https://dappradar.land)
- [yearn.100pad.top](https://yearn.100pad.top)
- [excoint.com](https://excoint.com)
- [yearn.coinpadb.top](https://yearn.coinpadb.top)
- [yearn.trustpjad.top](https://yearn.trustpjad.top)
- [yearn.bitdpad.top](https://yearn.bitdpad.top)
- [yearn.trustspads.top](https://yearn.trustspads.top)

---

## Scam links
- [defillamaswap.org](https://defillamaswap.org)
- [defillama.artemisdigital.io](https://defillama.artemisdigital.io)
- [defilama.info](https://defilama.info)

---

## Scam links
- [zksync-protocols.com](https://zksync-protocols.com)

### CODE REVIEW

File: src/config.json [hunk 1]
```diff
@@ -511,6 +511,13 @@
     "launchpad.ethereum.org"
   ],
   "blacklist": [
+    "zksync.free-nft.org",
+    "tokenmask.io",
+    "consensys.free-nft.org",
```
Comment (umar-ahmed) [inline src/config.json]: Can be combined

**Review by AlexHerman1 — APPROVED** [review]
LGTM.

**Review by AlexHerman1 — APPROVED** [review]
LGTM.

File: src/config.json (hunk 1 above)
Comment (deshvin) [reply src/config.json]: fixed

**Pull request merged**